doc/syntax.bnf

```diff
@@ -157,6 +158,8 @@ reserved "compute"
   | "set" "debug" RE("[-+][a-zA-Z]+")
   | "set" "builtin" <string_lit> "≔" <qident>
   | "set" "infix" {"left" | "right"}? <float_lit> <string_lit> "≔" <qident>
+  | "set" "prover" <ident>
+  | "set" "prover_limit" <nat_lit>
```

Comment:
Maybe rename `"prover_limit"` into `"timeout"`?

Reply:
Only `timeout` could be confused with the type checking of `lambdapi`. I think it is better to specify which timeout like : `prover_timeout`.

Reply:
I agree with Yacine: prover_timeout.

Reply:
Changed in 8f146ae2e9014e